src/test/java/com/redhat/devtools/lsp4ij/features/completion/QuteCompletionTest.java

```diff
@@ -24,70 +25,73 @@ public QuteCompletionTest() {
-                                   },
-                                   "filterText": "for",
-                                   "insertTextFormat": 2,
-                                   "insertTextMode": 2,
-                                   "textEdit": {
-                                     "range": {
-                                       "start": {
-                                         "line": 1,
-                                         "character": 4
-                                       },
-                                       "end": {
-                                         "line": 1,
-                                         "character": 6
-                                       }
-                                     },
-                                     "newText": "{#for ${1:item} in ${2:items}}\n\t{${1:item}.${3:name}}$0\n{/for}"
-                                   }
-                                 }
-                               ]
-                             }"""
-                ,
+                          "isIncomplete": false,
+                          "items": [
+                            {
+                              "label": "if",
+                              "kind": 15,
+                              "detail": "If section",
+                              "documentation": {
+                                "kind": "markdown",
+                                "value": "\\r\\n```\\r\\n{#if condition}\\n\\t\\n{/if}\\r\\n```\\r\\n\\r\\n\\r\\nSee [If section](https://quarkus.io/guides/qute-reference#if_section) for more information."
+                              },
+                              "filterText": "if",
+                              "insertTextFormat": 2,
+                              "insertTextMode": 2,
+                              "textEdit": {
+                                "range": {
+                                  "start": {
+                                    "line": 1,
+                                    "character": 4
+                                  },
+                                  "end": {
+                                    "line": 1,
+                                    "character": 5
+                                  }
+                                },
+                                "newText": "{#if ${1:condition}}\\n\\t$0\\n{/if}"
+                              }
+                            },
+                            {
+                              "label": "for",
+                              "kind": 15,
+                              "detail": "Loop section with alias",
+                              "documentation": {
+                                "kind": "markdown",
+                                "value": "\\r\\n```\\r\\n{#for item in items}\\n\\t{item.name}\\n{/for}\\r\\n```\\r\\n\\r\\n\\r\\nSee [Loop section](https://quarkus.io/guides/qute-reference#loop_section) for more information."

```

Comment:
It also looks like this was hand-edited from the original response because quite a bit of the required escaping was lost in the original version but has been restored properly here.